Makefile

```diff
@@ -10,7 +10,7 @@ pytest:
 
 cover-pytest: PYTEST_FLAGS += --cov=taskwiki
 cover-pytest: pytest
-	if [ "$$GITHUB_ACTIONS" ]; then coveralls || :; fi
+	if [ "$$GITHUB_ACTIONS" ]; then coveralls --service=github || :; fi
```

Comment:
Is this necessary? We already have `COVERALLS_SERVICE_NAME: github` in `.github/workflows/tests.yaml`.

Reply:
I think you're correct this might not fix the issue, I ported this fix from a different project, unaware we already had the env variable. It looks like coveralls.io is (still) simply down.

Reply:
I see.

(I can try to look into this, but I might be away from a computer for a couple more days.)

Reply:
Looks like the situation [is resolved now](https://status.coveralls.io).